magpie/setup/magpie-setup-project-zeppelin

```diff
@@ -224,11 +246,33 @@ fi
 
 ZEPPELIN_SPARK_SUBMIT_OPTS="${ZEPPELIN_SPARK_SUBMIT_OPTIONS}${ZEPPELIN_SPARK_SUBMIT_OPTIONS:+" "}--jars $zeppelin_jars"
 
-zeppelinclasspath=${spark_jars_path}/*:${CLASSPATH}
+for jar in $(ls ${spark_jars_path}/*.jar)
+do
+    zeppelinclasspath="${spark_jars_path}/${jar}:${zeppelinclasspath}"
+done
+
+for jar in $(ls ${ZEPPELIN_HOME}/lib/*.jar)
+do 
+    zeppelinclasspath="${ZEPPELIN_HOME}/lib/${jar}:${zeppelinclasspath}"
+done
+
+for jar in $(ls ${ZEPPELIN_HOME}/lib/interpreter/*.jar)
+do 
+    zeppelinclasspath="${ZEPPELIN_HOME}/lib/interpreter/${jar}:${zeppelinclasspath}"
+done
 
 # Magpie_find_conffile will set the 'pre' filenames
 Magpie_find_conffile "Zeppelin" ${ZEPPELIN_CONF_FILES:-""} "zeppelin-env.sh" "pre_zeppelinenvsh"
-Magpie_find_conffile "Zeppelin" ${ZEPPELIN_CONF_FILES:-""} "zeppelin-site.xml" "pre_zeppelinsitexml"
+
```

Comment:
I didn't look closely, but why the new zeppelin-site-0.8.0.xml file?  Just adding new fields to the old zeppelin-site.xml isn't possible?

Reply:
It is possible. The only conflict I see is that there are a ton of new interpreters added. In addition, there are a lot of new properties added and I wasn't sure how this would impact previous versions that don't have them. I thought it might be safer to just use the 0.8.0 config. I can merge/test them if you would like.

Reply:
I'm just a little confused on how this works.  Does zeppelin 0.8.0 do auto finding / detecting of things where prior versions did not?

For example, you have a lot of changes setting ```zeppelinclasspath```, but ```zeppelinclasspath``` is only used when substituting ```ZEPPELINCLASSPATH``` in conf files.  And ```ZEPPELINCLASSPATH``` is only used in ```zeppelin-env.sh```, not your new ```zeppelin-env-0.8.0.sh```.  So I'm not sure how this works.

Reply:
I will remove zeppelin-env-0.8.0.sh. I will also test if the 0.8.0 version of the xml file works with the previous version. If so I will update that.